src/client/util.js

```diff
@@ -16,19 +17,21 @@ export function isAndroidApp () {
   return typeof AndroidBridge !== 'undefined'
 }
 
-/* usage:
- * var connectWebViewJavascriptBridge = require('webViewJavascriptBridge')
- * connectWebViewJavascriptBridge(function(bridge) {
- *   bridge.send(...)
- * }
- */
-export function connectWebViewBridge (callback) {
-  if (window.WebViewJavascriptBridge) {
-    callback(window.WebViewJavascriptBridge)
-  } else {
-    document.addEventListener('WebViewJavascriptBridgeReady', function () {
-      callback(window.WebViewJavascriptBridge)
-    }, false)
+export function calliOSBridge (message, callback) {
```

Comment:
OK, I'll defer to you here.

On Fri, May 27, 2016 at 4:32 PM Robbie Carlton <notifications@github.com>
wrote:

> In src/client/util.js
> <https://github.com/Hylozoic/hylo-redux/pull/56#discussion_r64978086>:
>
> > @@ -16,19 +17,21 @@ export function isAndroidApp () {
> >    return typeof AndroidBridge !== 'undefined'
> >  }
> >
> > -/* usage:
> > - * var connectWebViewJavascriptBridge = require('webViewJavascriptBridge')
> > - * connectWebViewJavascriptBridge(function(bridge) {
> > - *   bridge.send(...)
> > - * }
> > - */
> > -export function connectWebViewBridge (callback) {
> > -  if (window.WebViewJavascriptBridge) {
> > -    callback(window.WebViewJavascriptBridge)
> > -  } else {
> > -    document.addEventListener('WebViewJavascriptBridgeReady', function () {
> > -      callback(window.WebViewJavascriptBridge)
> > -    }, false)
> > +export function calliOSBridge (message, callback) {
>
> I vote to keep it as it is.
> I think we'll always want to have two different functions, one for Android
> and one for iOS, partly because Android has way richer js bridge
> functionality which makes the code in the android app simpler, but would be
> hard/defeat the point to reduce it to the same interface as the iOS bridge,
> and partly because there are times when we may want to call one and not the
> other.
>
> these changes won't impact the android situation. The Android bridge is
> the same and being called in the same way for Filepicker.
> There are likely some little bugs in the android app that need ironing out
> with the new release.
>
> —
> You are receiving this because you were mentioned.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/Hylozoic/hylo-redux/pull/56/files/f33e8b2391a48ce232e928328b1a60e41efb2fc5#r64978086>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAL0g_5Qa6LDFW1Y4WGeSSxwv4172zs6ks5qF38jgaJpZM4Io67i>
> .
>